### ISSUE TITLE
Global dock window

### ISSUE THREAD

**Issue by MaxThom** [zed-industries/zed#14263]
### Check for existing issues

- [X] Completed

### Describe the feature

Hi, 
there is a feature I would like to have for my workflow. I was wondering if it's possible to have a global window that we can toggle with a key combo. Like the ZoomIn and ZoomOut feature, but instead of being for the current active window, it's a global window. It means, we could have a our code open and when we want to run it, we can toggle that global window that lays over like the ZoomIn with a terminal or more tabs. I'm coding on a 13inch laptop and have a terminal open takes a lot of space. I saw something that look along those lines on the website main feature page, but it changed since then.

I also want to add that I am absolutely in love with Zed. I'm coming from neovim and vscode. I got an epic project to work on, an awesome new laptop and an epic code editor.

Thanks!

### If applicable, add mockups / screenshots to help present your vision of the feature

![globalwindow_zed](https://github.com/user-attachments/assets/eacb5856-745c-40da-80fe-fedeecb66437)

**Comment by notpeter** [comment]
Let me see if I understand.  You're looking for some way to have a single-window optimized view because you're working on a small 13inch display Not quite sure I follow the zoom-in/zoom-out / global analogy. Could you try and explain further?

How is this different from dragging a terminal to the top tab bar and hiding the left sidebar (`ctrl-b` / `cmd-b`) and having everything in the same main pane?

**Comment by MaxThom** [comment]
What I mean is a floating terminal or window like this! So similar style of the ZoomIn
![image](https://github.com/user-attachments/assets/944d91db-60f1-47d5-a492-b1996aa4b132)

There was  a section on the Website named Dock which gave me the impression that what it was. The website got updated recently and removed that section.

I decided to update the keybindings and add this to the editor context 
`"ctrl-j": "workspace::ToggleBottomDock"`
